### ISSUE TITLE
Searching by accessibility traits does not work correctly.

### ISSUE THREAD

**Issue by ximenean** [wix/Detox#1131]
#### Description
When searching for an element with the trait `header` I noticed that the Element matcher was searching for the trait `UIAccessibilityTraitImage`. 

It looks like the flags are set incorrectly in `src/ios/earlgreyapi/GREYMatchers.js` (https://github.com/wix/Detox/blob/master/detox/src/ios/earlgreyapi/GREYMatchers.js).

I found the following list which seems pretty accurate:
https://docs.microsoft.com/en-us/dotnet/api/UIKit.UIAccessibilityTrait

I'd be happy to submit a PR for this but I'd appreciate some guidance as to whether this change would have any knock-on impact (I assume the list was correct at one point) and given that [accessibility traits are being deprecated](https://docs.microsoft.com/en-us/dotnet/api/UIKit.UIAccessibilityTrait) whether it is better to leave it.

#### Steps to Reproduce

Search for an element by any trait other than `button` and `link` and you will see it is mismatched.

#### Detox, Node, Device, Xcode and macOS Versions

* Detox: 9.1.2
* React Native: 0.57.8
* Node: 8.12.0
* Device: iPhone X (iOS 12.1)
* Xcode: 10.1
* macOS: Mojave 10.14.3

#### Device and verbose Detox logs

I don't believe the logs add value for this issue but I can provide them if required.

Here is a snippet from the error trace for the following test:

```javascript
it("should show the client's name", async () => {
  await expect(
    element(by.text("Mr C R Green").and(by.traits(["header"])))
  ).toBeVisible()
})
```

    Failed: [Error: Error: Cannot find UI Element.
    Exception with Assertion: {
      "Assertion Criteria":  "assertWithMatcher:matcherForSufficientlyVisible(>=0.750000)",
      "Element Matcher":  "((!(kindOfClass('RCTScrollView')) && ((((kindOfClass('UILabel') || kindOfClass('UITextField') || kindOfClass('UITextView')) && hasText('Mr C R Green')) || (kindOfClass('RCTTextView') && an object with accessibilityLabel "Mr C R Green")) && ((respondsToSelector(isAccessibilityElement) && isAccessibilityElement) && accessibilityTraits: UIAccessibilityTraitImage))) || (((kindOfClass('UIView') || respondsToSelector(accessibilityContainer)) && parentThatMatches(kindOfClass('RCTScrollView'))) && ((kindOfClass('UIView') || respondsToSelector(accessibilityContainer)) && parentThatMatches(((((kindOfClass('UILabel') || kindOfClass('UITextField') || kindOfClass('UITextView')) && hasText('Mr C R Green')) || (kindOfClass('RCTTextView') && an object with accessibilityLabel "Mr C R Green")) && ((respondsToSelector(isAccessibilityElement) && isAccessibilityElement) && accessibilityTraits: UIAccessibilityTraitImage))))))",
      "Recovery Suggestion":  "Check if the element exists in the UI hierarchy printed below. If it exists, adjust the matcher so that it accurately matches element."
    }

    Error Trace: [
      {
        "Description":  "Interaction cannot continue because the desired element was not found.",
        "Error Domain":  "com.google.earlgrey.ElementInteractionErrorDomain",
        "Error Code":  "0",
        "File Name":  "GREYElementInteraction.m",
        "Function Name":  "-[GREYElementInteraction matchedElementsWithTimeout:error:]",
        "Line":  "124"
      }
    ]
